autofs-5.0.7 - fix init script status return

From: Ian Kent <ikent@redhat.com>

The patch that added the piddir to configure to fix incorrect init
script status() function returns failed to actually return the value
to the user.
---

 CHANGELOG             |    1 +
 redhat/autofs.init.in |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)


--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -119,6 +119,7 @@
 - fix nobind man page description.
 - add piddir to configure.
 - fix submount offset delete.
+- fix init script status return.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/redhat/autofs.init.in
+++ autofs-5.0.5/redhat/autofs.init.in
@@ -172,7 +172,7 @@ RETVAL=0
 case "$1" in
 	status)
 		status -p @@autofspiddir@@/autofs.pid -l autofs $prog
-		exit 0;
+		exit $?;
 		;;
 	usage)
 		usage_message
